scripts/docker_start_user.sh

```diff
@@ -74,6 +74,9 @@ function grant_device_permissions() {
     # setup camera device
     [ -e /dev/camera/obstacle ]      && chmod a+rw /dev/camera/obstacle
     [ -e /dev/camera/trafficlights ] && chmod a+rw /dev/camera/trafficlights
+
+    # setup audio device
+    [ -e /dev/snd ] && adduser "$1" audio
```

Comment:
```
adduser "$1" audio
usermod -a -G audio  "$1"
```
Although they are both OK,  I prefer `usermod -a -G audio $1` here for readability.

Reply:
Sure, I just tested `usermod -a -G audio  "$1"` and it worked too. I have fixed it.